ios/forward/forward.go

```diff
@@ -16,28 +16,54 @@ type iosproxy struct {
 	deviceConn ios.DeviceConnectionInterface
 }
 
+type ConnListener struct {
+	listener net.Listener
+	quit     chan interface{}
+}
+
 // Forward forwards every connection made to the hostPort to whatever service runs inside an app on the device on phonePort.
-func Forward(device ios.DeviceEntry, hostPort uint16, phonePort uint16) error {
+func Forward(device ios.DeviceEntry, hostPort uint16, phonePort uint16) (*ConnListener, error) {
 	log.Infof("Start listening on port %d forwarding to port %d on device", hostPort, phonePort)
 	l, err := net.Listen("tcp", fmt.Sprintf("0.0.0.0:%d", hostPort))
 	if err != nil {
-		return err
+		return nil, fmt.Errorf("forward: failed listener with err: %w", err)
+	}
+	cl := &ConnListener{
+		listener: l,
+		quit:     make(chan interface{}),
 	}
 
-	go connectionAccept(l, device.DeviceID, phonePort)
+	go connectionAccept(cl, device.DeviceID, phonePort)
+
+	return cl, nil
+}
+
+func (cl *ConnListener) Close() error {
```

Comment:
Lets also try to document exported methods 🙂 
```suggestion
// Close stops listening on the host port for the forwarded connection
func (cl *ConnListener) Close() error {
```